ext/openssl/tests/openssl_cms_decrypt_basic.phpt

```diff
@@ -0,0 +1,66 @@
+--TEST--
+openssl_cms_decrypt() tests
+--SKIPIF--
+<?php if (!extension_loaded("openssl")) print "skip"; ?>
+--FILE--
+<?php
+$infile = __DIR__ . "/cert.crt";
+$privkey = "file://" . __DIR__ . "/private_rsa_1024.key";
+$encrypted = tempnam(sys_get_temp_dir(), "ssl");
+if ($encrypted === false)
+    die("failed to get a temporary filename!");
+$outfile = tempnam(sys_get_temp_dir(), "ssl");
+if ($outfile === false) {
+    unlink($outfile);
+    die("failed to get a temporary filename!");
+}
+
+$single_cert = "file://" . __DIR__ . "/cert.crt";
+$headers = array("test@test", "testing openssl_cms_encrypt()");
+$wrong = "wrong";
+$empty = "";
+
+openssl_cms_encrypt($infile, $encrypted, $single_cert, $headers);
+var_dump(openssl_cms_decrypt($encrypted, $outfile, $single_cert, $privkey));
```

Comment:
might be also good to verify that the expected data is in the outfile...

Reply:
we can't do a direct compare, but we can validate that there is at least something there.

Reply:
So this is not resolved, right?